Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Base image for container
+FROM golang:alpine3.18
```

Comment:
No debéis usar una versión específica, y menos si no es la última.

Reply:
No entiendo muy bien, te refieres a que tendría que haber puesto `golang:alpine`?

Reply:
Nunca te voy a decir que pongas esto o lo otro, la decisión la tienes que tomar tú. Pero [si miras en los errores frecuentes](https://github.com/JJ/IV-/blob/master/errores/errores-objetivo-5.md) dice:

> De la misma forma, basarse sólo en el tamaño puede conducir a elegir imágenes cuya velocidad a la hora de ejecutar los tests no sea óptima.

En tu caso no has testeado la velocidad de los tests, pero el punto sigue siendo que elegir *la versión* de la imagen base en función del tamaño crea deuda técnica, porque al no ser la última versión es algo que hay que comprobar periódicamente a ver si hay errores de seguridad o de otro tipo (no es que tengan mucha importancia en una imagen de test, pero en todo caso puede tenerlo si luego se extiende a una de despliegue); en todo caso no es justificable usar una versión del SO base en vez de simplemente la última versión, que tendrá todo corregido y además se actualizará periódicamente cada vez que se cree la imagen.

Reply:
Entiendo entonces que lo ideal si al final usase la oficial (no lo hago, arreglado aquí [3be7b2f](https://github.com/diagmatrix/maybe-better-maybe-worse/pull/29/commits/3be7b2f4d4289b103295648749adb7b8c9005824)), ¿tendría que usar `golang:latest` independientemente del SO base en el que esté construida? 

Por curiosidad, ¿ese SO puede cambiar? Las veces que he usado Docker anteriormente, he usado imágenes que sólo estaban en base a un SO, entonces no sé si eso puede pasar.